server/container/guice/cassandra-guice/pom.xml

```
@@ -273,6 +278,124 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-jbpm</id>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <url>https://github.com/glowroot/glowroot/releases/download/v0.13.4/glowroot-0.13.4-dist.zip</url>
+                            <unpack>true</unpack>
+                            <outputDirectory>${project.build.directory}</outputDirectory>
+                            <md5>93d472281a67ab8404033a88280c23c9</md5>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-glowroot-resources</id>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <outputDirectory>${basedir}/target/glowroot</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/glowroot</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <configuration>
+                    <from>
+                        <image>adoptopenjdk:11-jdk-hotspot</image>
```

```suggestion
                        <image>gcr.io/distroless/java:11</image>
```
along with https://martinheinz.dev/blog/49 for debugging ?

applies to all pom.xml which use jib :D

---

I am unsure we should change the running images. What are the pros and cons?

Maybe we need more discussion and you could propose that change (and related fixes like JVM option passing, glowroot). Likely not a drop in replacement.

---

pros: 
- less attack surface in the running container as there is way less  stuff installed that can be used to exploit
- much much lighter images (some report 75% reduction)
cons: 
- forget docker exec distroless containers only contain strictly necessary (hence ephemeral containers)